src/matrix4.js

```diff
@@ -153,9 +153,9 @@ export default class Matrix4 extends MathArray {
   }
   /* eslint-enable no-multi-spaces, brace-style, no-return-assign */
 
-  copy(array) {
-    return this.setColumnMajor(...array);
-  }
+  // copy(array) {
+  //   // return this.setColumnMajor(...array);
+  // }
```

Comment:
This removal is interesting, did you see an improvement from it?

Also (unrelated to this PR), it seems like `copy`, `set`, and `setColumnMajor` are all quite similar. Do we need all of them? And if we do, might be simpler to write them all in terms of `MathArray.copy`, e.g. 
```
set(...args) {
    return this.copy(args);
}
```

Reply:
> This removal is interesting, did you see an improvement from it?

Yes, it did make a difference. If you remove the `testDone` call in `test/bench/browser.js` call, and run `yarn test bench-browser` the benchmarks will hot-reload and you can play around with changes like these.

> it seems like `copy`, `set`, and `setColumnMajor` are all quite similar. Do we need all of them?

Good observation. No I don't think we need them all. I did want to emphasize that we store data in column major format but I am not sure the current setters help the users much. And even if we keep them, the rest of the setters should be optimized and don't need to use them.

Adding a set of benchmarks for all the setters would be really helpful, and then do a pass an optimizing them "to-the-bone". Maybe we could write a function that takes a class and adds benchmarks for all the common `math-array` methods for that class?

Reply:
Also math.gl doesn't have any examples. One idea could be to have the benchmarks as an example so that users could run them on their browser and see the results...